openpype/settings/entities/schemas/projects_schema/schemas/schema_global_publish.json

```diff
@@ -18,6 +18,27 @@
                 }
             ]
         },
+        {
+            "type": "dict",
+            "collapsible": true,
+            "key": "CollectSceneVersion",
+            "label": "Collect Version from Workfile",
+            "is_group": true,
+            "children": [
+                {
```

Comment:
I don't think we should give option to change `hosts`. At least they should be limited to hosts that make sense.